src/app/components/profile/profile.html

```diff
@@ -255,8 +260,8 @@
                 <div class="panel-footer" ng-if="vm.skills.length > 15">
                     <dir-pagination-controls></dir-pagination-controls>
                 </div>
-                <div class="panel-body" ng-if="vm.skills.length === 0" ng-click="vm.showModal(vm.skillsModal, vm.skills)" ng-show="vm.isAllowed('canEditUser') || vm.isSelf(vm.user._id)">
-                    You have not added any skills yet. To add a skill, press the <i class="fa fa-edit"></i> button in the top right corner.
```

Comment:
Please remove the newline at row 283 :) I know it's not yours but since you are already there